vivliostyle-cli.md

```diff
@@ -0,0 +1,325 @@
+
+Example:
+```
+vivliostyle build paper.html -s A4 -o paper.pdf
+vivliostyle build letter.html -s letter -o letter.pdf
+vivliostyle build slide.html -s 10in,7.5in -o slide.pdf
+```
+
+### Specify a Web URL
+
+In addition to a local HTML file, you can also specify a Web URL.
+
+Example:
+```
+vivliostyle build https://vivliostyle.github.io/vivliostyle_doc/samples/gutenberg/Alice.html -s A4 -o Alice.pdf
+```
+
+## Generate PDF from EPUB
+
+When an EPUB file is specified with the `vivliostyle build` command, a PDF file will be output as a result of typesetting from the EPUB.
+
+Example:
+```
+vivliostyle build ebpaj-sample.epub -s A5 -o ebpaj-sample.pdf
+```
+
+### Generate PDF from unzipped EPUB
+
+To generate a PDF from an unzipped EPUB, specify the EPUB's OPF file.
+
+Example:
+```
+unzip ebpaj-sample.epub
+vivliostyle build item/standard.opf -s A5 -o ebpaj-sample.pdf
+```
+
+## Generate PDF from Markdown
+
+When a Markdown file is specified with the `vivliostyle build` command, a PDF file will be output as a result of typesetting from the Markdown.
+
+Example:
+```
+vivliostyle build manuscript.md -s A4 -o paper.pdf
+```
+
+### About the VFM (Vivliostyle Flavored Markdown)
+
+For more information on the Markdown notation available in Vivliostyle CLI, refer to [VFM: Vivliostyle Flavored Markdown](https://vivliostyle.github.io/vfm/#/).
+
+### Specify a CSS style sheet
+
+The `-T` (`--theme`) option can be used to specify a CSS file.
+
+Example:
+```
+vivliostyle build manuscript.md -T style.css -o paper.pdf
+```
+
+## Preview the typesetting results
+
+The `vivliostyle preview` command can be used to preview the result of typesetting in the browser.
+
+Example:
+```
+vivliostyle preview manuscript.md -T style.css
+```
+
+## About Vivliostyle Themes
+
+- [Vivliostyle Themes](https://vivliostyle.github.io/themes/)
+
+### Find themes
+
+To find the themes published as npm packages, search for the keyword "vivliostyle-theme" in [npm](https://www.npmjs.com/):
+
+- [List of Themes (npm)](https://www.npmjs.com/search?q=keywords%3Avivliostyle-theme)
+
+### Install a theme
+
+To use a theme published as a npm package in Vivliostyle CLI, you need to install it.
+
+Example:
+```
+npm install @vivliostyle/theme-techbook
+```
+
+### Using the Create Book
+
+A easier way to use Vivliostyle Themes is to use Create Book. See [Create Book](create-book).
+
+## Configuration file vivliostyle.config.js
+
+To combine multiple article or chapter files into a single publication, use the configuration file. When running the `vivliostyle build` or `vivliostyle preview` command, the configuration file `vivliostyle.config.js` will be used if it exists in the current directory.
+
+### Create configuration file
+
+You can create a configuration file `vivliostyle.config.js` with the following command:
+
+```
+vivliostyle init
+```
+
+This will create `vivliostyle.config.js` in the current directory. Edit it and configure the settings.
```

Comment:
```suggestion
This will create `vivliostyle.config.js` in the current directory. The configuration file is written in JavaScript, and various settings can be changed by editing it.
```